pypsa/components.py

```diff
@@ -465,6 +475,13 @@ def meta(self, new):
             raise TypeError(f"Meta must be a dictionary, received a {type(new)}")
         self._meta = new
 
+    @property
+    def crs(self):
+        """
+        Coordinate reference system of the network.
+        """
+        return self.shapes.crs
```

Comment:
This overlaps with `n.srid`, which is essentially the same. What do we do with this? Deprecate `n.srid` and map `n.crs` onto it in the mean time?

I am wondering what I have to do to change the projection of my `n.shapes`.

Probably, `n.shapes = n.shapes.to_crs("3035")` would work.

Maybe, a utility function to do the projection of `n.shapes` automatically if you say one of:

```py
n.crs = 3035

n.crs = "EPSG:3035"

crs = ccrs.EqualEarth()
n.crs = crs

crs = ccrs.EqualEarth()
n.crs = crs.proj4_init
```

Reply:
One problem I see at the moment, is that srid is actually referring to the coordinates of n.buses. So, in the current implementation, crs is referring to the projection of the geometries only. I am updating the docstrings accordingly. For later (in another PR), I would propose to make a smooth transition to replace srid & crs by crs only which then refers to geometries and coordinates. When setting the crs, the coordinates of the buses should then be updated as well. Does this make sense?

Reply:
Sounds good! The `@setter` function for reprojection would still be nice.

Reply:
done